source/help/getting-started/organizing.rst

```diff
@@ -42,151 +42,150 @@ As you grow further, you can combine the hierarchies, with formats like ``[SUB-T
 
 .. tip :: Shorten words, particularly categories, by removing vowels, endings and redundant letter sounds. Example: Turn "Marketing" into "Mrkt", and "Project" into "Prjt".
 
-Good naming can take a team up to several thousand channels without causing significant confusion. Eventually every organization hits a limit and an additional team might need to be created on the server to accommodate a very large number of channels. 
+Good naming can take a team up to several thousand channels without causing significant confusion. Eventually every organization hits a limit and an additional team might need to be created on the server to accommodate a very large number of channels.
 
-Navigating to channels using the keyboard 
+Navigating to channels using the keyboard
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Another reason why channel names in Mattermost are limited to 22 characters is the common use of keyboard shortcuts to jump quickly to channels. 
+Another reason why channel names in Mattermost are limited to 22 characters is the common use of keyboard shortcuts to jump quickly to channels.
 
-User use CTRL+K to bring up a dialog where they can quickly type in the first few letters of a channel, which triggers auto-complete, and hitting ENTER to jump to the channel directly. 
+User use CTRL+K to bring up a dialog where they can quickly type in the first few letters of a channel, which triggers auto-complete, and hitting ENTER to jump to the channel directly.
```

Comment:
I changed it to this instead, in order to be more consistent with the rest of the doc:
>Users can use CTRL+K to bring up a dialog where they can quickly type in the first few letters of a channel, which triggers auto-complete. Pressing ENTER jumps to the channel directly.

I'm rewriting this topic soon anyway.